modules/security_hub/variables.tf

```diff
@@ -1,3 +1,8 @@
+variable "account_id" {
+  type        = string
+  description = "AWS Account ID"
+}
```

Comment:
What is this for? There is no reference to this variable anywhere else.

Reply:
it is referenced in `topic_arn  = "arn:aws:sns:${var.region}:${var.account_id}:aws-controltower-AggregateSecurityNotifications"`, **line 34** in **modules/security_hub/main.tf**